src/Console/Command/HelpCommand.php

```diff
@@ -508,6 +509,10 @@ function (FixerOptionInterface $optionA, FixerOptionInterface $optionB) {
                             $line .= 'required';
                         }
 
+                        if ($option instanceof AliasedFixerOption) {
+                            $line .= '; alias: <comment>'.$option->getAlias().'</comment>';
```

Comment:
don't we need an escape on `$option->getAlias()`? maybe wrap this in a `sprintf` as well

Reply:
I was following line 486, no escaping used there